BDKSwiftExampleWallet/Extensions/BDK+Extensions/Transaction+Extensions.swift

```diff
@@ -6,17 +6,18 @@
 //
 
 import BitcoinDevKit
+import Foundation
 
 extension Transaction {
     var transactionID: String {
-        return self.computeTxid()
+        return "\(self.computeTxid())"
     }
 }
 
 //#if DEBUG
 extension Transaction {
     static var mock = try? Transaction(
-        transactionBytes: String.mockTransactionHex.hexStringToByteArray()
+        transactionBytes: Data(String.mockTransactionHex.hexStringToByteArray())
```

Comment:
I think similar-ish to https://github.com/bitcoindevkit/BDKSwiftExampleWallet/pull/297/files#r2124988471 ?

new expectation is `Data` as argument type